e2e/select_test.go

```diff
@@ -486,6 +523,72 @@ func TestSelectCommand(t *testing.T) {
 	}
 }
 
+func TestSelectCommandEmptyBucket(t *testing.T) {
+	t.Parallel()
+
+	const (
+		region      = "us-east-1"
+		accessKeyID = "minioadmin"
+		secretKey   = "minioadmin"
+
+		query = "SELECT * FROM s3object s"
+	)
+
+	endpoint := os.Getenv(s5cmdTestEndpointEnv)
+	if endpoint == "" {
+		t.Skipf("skipping the test because %v environment variable is empty", s5cmdTestEndpointEnv)
+	}
+
+	type testcase struct {
+		name          string
+		cmd           []string
+		expectedValue string
```

Comment:
```suggestion
		expectError bool
```
__Current cases__ require no `expectedValue`. When there is a case that will require `expectedValue`, the author can refactor accordingly. The assertions of these cases are there are no errors, refactoring the runner accordingly would be better.